src/raftstore/store/peer.rs

```diff
@@ -203,7 +203,13 @@ impl Peer {
 
         let store_id = store.store_id();
         let sched = store.snap_scheduler();
-        let ps = try!(PeerStorage::new(store.engine(), &region, sched));
+        let tag = format!("[region {}]", region.get_id());
```

Comment:
If ok, we may write log like `info!("{} became leader at term {}", self.tag, self.term);`, and I think it is not clear for user to see this log, who became the leader? the tag? 

But I know that put id in tag can reduce code, and I did it before.
/cc @ngaut